### PR TITLE
Update README to markdown format

### PR DESCRIPTION
**Pull request by sathnaga**
Update Readme and change to markdown format
for better view in github.

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by sathnaga** [comment]
can be reviewed better by https://github.com/open-power-host-os/tests/blob/f51a8b50c33461c7bc1b78b44c0dad5bfb03b27e/README.md

File: README.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+### Argument Details:
+1. `--bootstrap`:
+
+    > Use this option to bootstrap the environment so that all of the required repositories (avocado, avocado-vt, avocado-misc-tests) are downloaded, installed, and configured.
+    >
+    > Example command to bootstrap the test suite with only host tests.
+    >
+    > `./avocado-setup.py --bootstrap`
+
```
Comment (narasimhan-v) [inline README.md:114]: Is this needed ?

Comment (sathnaga) [reply README.md:114]: it is good to be in detail, difference between --enable-kvm and default...

Comment (sathnaga) [reply README.md:114]: if you are asking about additional line, yes it does in markdown

Comment (sathnaga) [reply README.md:114]: infact next line is missing it, will add nw...

**Pull request merged**